RDKShell/RDKShell.cpp

```diff
@@ -197,6 +197,9 @@ bool sForceResidentAppLaunch = false;
 static bool sRunning = true;
 bool needsScreenshot = false;
 
+std::mutex suspendedOrHibernatedAppsMutex;
```

Comment:
Might be good to use the #ifdef HIBERNATE_SUPPORT_ENABLED everywhere we use suspendOrHibernateApps array and mutex.